.github/workflows/release.yml

```diff
@@ -90,3 +90,10 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
           npx nx run-many -t=deploy --exclude=plugin-lighthouse,nx-plugin
+      - name: Collect and Upload Code PushUp report
```

Comment:
In the [CI pipeline](https://github.com/code-pushup/cli/blob/main/.github/workflows/ci.yml#L188) where the reports are also gathered, there is an additional step for creating an artifact with the reports.
Is that not needed here? We should keep the workflows consistent.

Reply:
I can add it too. It is not a big problem, I think

Reply:
Yeah, why not :slightly_smiling_face: It can be copy-pasted from the other workflow: https://github.com/code-pushup/cli/blob/main/.github/workflows/ci.yml#L188-L192